src/net_processing.cpp

```diff
@@ -3536,14 +3536,6 @@ bool PeerLogicValidation::CheckIfBanned(CNode* pnode)
 bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& interruptMsgProc)
 {
     const CChainParams& chainparams = Params();
-    //
-    // Message format
-    //  (4) message start
-    //  (12) command
-    //  (4) size
-    //  (4) checksum
-    //  (x) data
-    //
```

Comment:
Why remove this comment block?

Reply:
Net Processing is no longer aware of the format of the header.  That's all dealt with in net, so that net processing can just process the payload.

Reply:
3a9d277 Is there somewhere suitable this documentation can be placed in net.h/cpp rather than be tossed out?